test/v6/fixtures/no-exposed-subjects/default/fixture.ts.lint

```diff
@@ -0,0 +1,48 @@
+import * as Rx from 'rxjs';
+
+const variable = new Rx.Subject<void>();
+
+class Mock {
+  private submitSubject$ = new Rx.Subject<void>();
+
+  private readonly rSubmitSubject$ = new Rx.Subject<void>();
+
+  public property$ = new Rx.Subject<void>();
+         ~~~~~~~~~            [Subject 'property$' must be private.]
+
+  protected property2$ = new Rx.Subject<void>();
+            ~~~~~~~~~~           [Subject 'property2$' must be private.]
+
+  anyPropertySubject$ = new Rx.Subject<void>();
+  ~~~~~~~~~~~~~~~~~~~         [Subject 'anyPropertySubject$' must be private.]
+
+  readonly public rPropertySubject$ = new Rx.Subject<void>();
```

Comment:
My understanding is that this is invalid syntax: the `readonly` modifier must be placed *after* the access modifier.